packages/slice-machine/src/models/slice/reducer.ts

```diff
@@ -202,16 +203,34 @@ export function reducer(
   return {
     ...result,
     isTouched: (() => {
```

Comment:
Here's what I don't get, if we have a version to compareVersions
- 1, it must be a bit heavy so I would do it only once.
- 2, why isn't "isTouched" a property instead of a function